cmd/frontend/graphqlbackend/textsearch.go

```diff
@@ -788,6 +795,32 @@ func searchFilesInRepos(ctx context.Context, args *search.TextParameters) (res [
 	return flattened, common, nil
 }
 
+const maxUnindexedRepoRevSearchesPerQuery = 200
+
+// limitSearcherRepos imposes a limit on the number of repo@revs that would go
+// to the unindexed searcher codepath, because sending this many requests to
+// searcher would otherwise result in the system and network being flooded with
+// requests that merely result in timeouts and take a long time to complete.
```

Comment:
```suggestion
// limitSearcherRepos limits the number of repo@revs searched by the
// unindexed searcher codepath. Sending many requests to
// searcher would otherwise cause a flood of system and network requests that
// result in timeouts or long delays.
```

Didn't format, but attempt at making terser comment and fit in 3 lines maybe :-)